source/expressions.tex

```diff
@@ -4672,11 +4673,22 @@
 The type of the result is \keyword{bool}.
 
 \pnum
+% FIXME: The removed term "ones' complement" remains in indexes;
+% should we fix this? Change to "bitwise complement" perhaps?
+% Similarly for "two's complement" which is only introduced in a footnote.
 \indextext{signed integer representation!ones' complement}%
```

Comment:
Let's delete the first index entry, but the second term "one's complement" is not about representation, but about the actual _operation_: `~x` is the ones' complement of `x`.

Reply:
See fixup commit.